secp256k1-sys/build.rs

```diff
@@ -59,6 +59,8 @@ fn main() {
 
     // secp256k1
     base_config.file("depend/secp256k1/contrib/lax_der_parsing.c")
+               .file("depend/secp256k1/src/precomputed_ecmult_gen.c")
+               .file("depend/secp256k1/src/precomputed_ecmult.c")
```

Comment:
note: In upstream, we compile these to a separate object which is then linked to the actual library code. This saves compilation time in the common case that the code has changed but tables haven't. But it's probably not worth the hassle here because we never touch the C code here, not even during development.